### PR TITLE
ARUHA-671 Remove part of timelines migration code 

### PR DESCRIPTION
**Pull request by antban**
ARUHA-671 Use timelines everywhere, remove migration code

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/subscription/zk/AbstractZkSubscriptionClient.java [hunk 1]
```diff
@@ -122,48 +115,28 @@ public final void deleteSubscription() {
...
-        }
     }
 
     @Override
-    public final void fillEmptySubscription(final Collection<SubscriptionCursorWithoutToken> cursorsIn) {
+    public final void fillEmptySubscription(final Collection<SubscriptionCursorWithoutToken> cursors) {
         try {
-            final Collection<SubscriptionCursorWithoutToken> cursors;
-            // Delete root subscription node
+            // Delete root subscription node, if it was erroneously created
```
Comment (rcillo) [inline src/main/java/org/zalando/nakadi/service/subscription/zk/AbstractZkSubscriptionClient.java]: When should this happen?

**Comment by rcillo** [comment]
👍

File: src/main/java/org/zalando/nakadi/service/subscription/zk/AbstractZkSubscriptionClient.java (hunk 1 above)
Comment (antban) [inline src/main/java/org/zalando/nakadi/service/subscription/zk/AbstractZkSubscriptionClient.java]: When initialization didn't finish (for example initialization started, but then someone killed nakadi with -9)

**Comment by antban** [comment]
:+1:

**Pull request merged**